src/common/CMakeLists.txt

```diff
@@ -1,15 +1,14 @@
 add_subdirectory(binaries_op)
-add_subdirectory(bzip2_op)
```

Comment:
Why are you removing this library?

Reply:
Same about `file_op.h`. If these two libraries are not needed, remove their code.